roles/installer/templates/statefulsets/postgres.yaml.j2

```diff
@@ -46,6 +46,29 @@ spec:
 {% endif %}
 {% if postgres_priority_class is defined %}
       priorityClassName: '{{ postgres_priority_class }}'
+{% endif %}
+{% if init_postgres_extra_commands %}
+      initContainers:
+        - name: init
+          image: '{{ _postgres_image }}'
+          imagePullPolicy: '{{ image_pull_policy }}'
+          securityContext:
+            runAsUser: 0
+          command:
+            - /bin/sh
+            - -c
+            - |
+              {{ init_postgres_extra_commands | indent(width=14) }}
+          volumeMounts:
+            - name: postgres-{{ supported_pg_version }}
+              mountPath: '{{ postgres_data_path | dirname }}'
+              subPath: '{{ postgres_data_path | dirname | basename }}'
+{% if postgres_extra_volume_mounts -%}
+            {{ postgres_extra_volume_mounts | indent(width=12, first=True) }}
+{% endif %}
+{% endif %}
+{% if init_container_extra_volume_mounts -%}
+            {{ init_container_extra_volume_mounts | indent(width=12, first=True) }}
```

Comment:
this is not in volumes. I think this needs to be removed.

Reply:
@hhk7734 
I think this is intended, since any other `*_extra_volume_mounts` are designed to use with single `extra_volumes`: https://ansible.readthedocs.io/projects/awx-operator/en/latest/user-guide/advanced-configuration/custom-volume-and-volume-mount-options.html

Reply:
If `init_container_extra_volume_mounts` is designed to use with `extra_volumes`, it need to add `extra_volumes` to volumes in this template.

Reply:
Oh, sorry, it looks like `postgres_extra_volumes` is used for the additional PSQL volumes instead of `extra_volumes`.
I don't see it mentioned in the documentation. This is a different issue: https://github.com/ansible/awx-operator/pull/1799/files#diff-57afb141659730e9bae27cd3ea91bfe5b1709a2569d98759221305423d544c03L128-L131

Reply:
I don't think that `init_container_extra_volume_mounts` block belongs here.  The `postgres_extra_volume_mounts` parameter is sufficient.